client/services/PlanningStoreService.ts

```diff
@@ -207,7 +207,9 @@ export class PlanningStoreService {
                     ),
                     ingest: {
                         providers: get(data, 'ingest', [])
-                            .filter((p) => get(p, 'content_types', []).indexOf(ITEM_TYPE.EVENT) !== -1)
+                            .filter((p) => (
+                                get(p, 'content_types', []) === ITEM_TYPE.EVENT || ITEM_TYPE.PLANNING)
```

Comment:
This won't filter any of the source providers, because ` || ITEM_TYPE.PLANNING` statement is run separately and always returns `true`.

You'd need to check the `ITEM_TYPE.[EVENT|PLANNING]` existence in `content_types` separately

Reply:
Can we use this as my second suggestion?, I think its works.
```
.filter((p) => (
           get(p, 'content_types', []).indexOf(ITEM_TYPE.EVENT) !== -1 ||
           get(p, 'content_types', []).indexOf(ITEM_TYPE.PLANNING) !== -1))
```

Reply:
@devketanpro Yes that will work